README.md

```diff
@@ -51,9 +51,12 @@ Generators:
 
 Once installation is complete, begin your journey!
 ```sh
-$ npm start
+$ node server.js
 ```
 
+`npm start` can also be used to start your application, but please note that it
```

Comment:
I don't think we need this caveat in the README. We can add it to the documentation somewhere (https://github.com/trailsjs/trails-docs)